Documents/public-api.md

```diff
@@ -0,0 +1,34 @@
+# HMDA Platform Public API
+
+This documenatation describes de public HMDA Platform HTTP API
+
+## Institutions
+
+### Search
+
+* `/institutions?domain=<domain>`
+
+   * `GET` - Returns a list of institutions filtered by their email domain. If none are found, an HTTP 404 error code (not found) is returned
+
+   Example response, with HTTP code 200:
+
+   ```json
+   [
+      {
+        "id": "0",
+        "name": "Bank 0",
+        "domains": ["test@bank0.com"],
+        "externalIds":[
```

Comment:
I think this discussion has come up before, and yes, there are cases where the id is undetermined for institutions if I remember correctly. @nickgrippin might know, if not this is a question for @Kibrael